pkg/kubelet/dockertools/docker_manager_linux.go

```diff
@@ -40,8 +40,13 @@ func getContainerIP(container *dockertypes.ContainerJSON) string {
 func getNetworkingMode() string { return "" }
 
 // Returns true if the container name matches the infrastructure's container name
-func containerProvidesPodIP(name *KubeletContainerName) bool {
-	return name.ContainerName == PodInfraContainerName
+func containerProvidesPodIP(containerName string) bool {
```

Comment:
Yeah, @alexbrand says each container in a Windows pod gets its own IP.  So containerProvidesPodIP() isn't quite the same as containerIsNetworked(), since one container is suppposed to supply the pod IP, but all of them are networked.